concent_api/core/tests/test_core_views.py

```diff
@@ -408,47 +389,6 @@ def test_send_should_return_http_400_if_task_to_compute_deadline_is_not_an_integ
             self.assertEqual(response_400.status_code, 400)
             self.assertIn('error', response_400.json().keys())
 
-    def test_receive_should_return_http_400_if_task_id_is_not_valid_string(self):
```

Comment:
Why is this test removed? Don't we still return HTTP 400 for invalid task IDs?

Reply:
On golem_messages v1.7.0. I am not able to set wrong task_id. By set I mean doing this
```message.task_id = 11```
It fails instantly with validation exception.

Reply:
Well, OK. We still need this test but we'll do it in a separate issue. Even if the message class won't let you set `task_id` to an invalid value via the property it's definitely still possible to manually craft such a message. We'll need to ensure that such messages are correctly detected as invalid and result in HTTP 400 instead of going through and making the code behave in an unexpected way. I see that this is a bigger problem though and will need to be addressed separately.